src/Digdir.Domain.Dialogporten.Domain/Dialogs/Entities/DialogUserType.cs

```diff
@@ -19,24 +19,19 @@ public enum Values
         /// </summary>
         Person = 1,
 
-        /// <summary>
-        /// Altinn 2 legacy system user ("virksomhetsbruker")
-        /// </summary>
-        LegacySystemUser = 2,
-
         /// <summary>
         /// Altinn 3 system user
         /// </summary>
-        SystemUser = 3,
+        SystemUser = 2,
```

Comment:
AbstractLookupTypes er ment til å være statisk.

Det å gjøre breaking changes på AbstractLookupTypes er ikke noe en skal ta lett på. Det som er i ferd med å skje her er at alle SeenLog entries som pekte på SystemUser (tidligere ID 3), vil nå peke på ServiceOwner (nye ID 3). Eller sagt med andre ord - ingen REFERSNSER endres når du gjør denne endringen, du skifter betydningen av hver ID. Det i seg selv er ikke så ille om du tar hensyn til det i migrasjonen. MEN

I tillegg bruker vi mange av våre AbstractLookupTypes i OuterFacingContract (OFC) fordi vi anser dem som statiske, som vil si når vi endrer disse kan vi ha breaking changes for konsumentene våre. Vi kan legge til verdier, og kanskje fjerne dem etter noe deprecated tid dersom de brukes av (OFC). Men syns ikke at vi skal legge oss til vane å shuffle rundt på IDene deres som dette.

Ser ikke ut som at akkurat denne brukes i OFC - så sånn sett good.

Reply:
Enig at dette i prinsippet er breaking, og er ikke noe vi skal gjøre uten videre når vi kommer i produksjon.